public/css/main.css

```diff
@@ -40,6 +40,16 @@ footer {
   height: 236px;
 }
 
+.carousel_image {
+  width: 100%;
+  height: 450px;
+}
+
+.carousel-caption h3 {
+  color:crimson;
```

Comment:
Do we want the description on top of the image? If so, what color do you make it? Some pictures are black vs white

Reply:
@dwilliamson5 this might make things easier: https://getbootstrap.com/docs/4.0/components/carousel/#with-captions

Reply:
Thank you. I did follow that but the color still seems to be an issue depending on the photo